packages/rs-sdk/src/platform/transition/put_document.rs

```diff
@@ -72,12 +72,16 @@ impl<S: Signer> PutDocument<S> for Document {
                 settings,
             )
             .await?;
+
+        let settings = settings.unwrap_or_default();
+
         let transition = DocumentsBatchTransition::new_document_creation_transition_from_document(
             self.clone(),
             document_type.as_ref(),
             document_state_transition_entropy,
             &identity_public_key,
             new_identity_contract_nonce,
+            settings.fee_multiplier.unwrap_or_default(),
```

Comment:
Shouldn't we use default constant here?

Reply:
no, you mixed up the default user multiplier with the system one.